lib/active_storage/blob.rb

```diff
@@ -48,7 +48,6 @@ def url_for_direct_upload(expires_in: 5.minutes)
     service.url_for_direct_upload key, expires_in: expires_in, content_type: content_type, content_length: byte_size
   end
 
-
```

Comment:
These are all intentional to group related method sets together. Whatever rule is creating this as a violation, let's drop it.